src/ValidationModels/penaltyValidation.js

```diff
@@ -40,19 +42,19 @@ const valueSchema = {
 	dateTime: Joi.number().integer().required(),
 	paymentCodeDateTime: Joi.number().integer(),
 	placeWhereIssued: Joi.string(),
-	officerID: Joi.string().required(),
+	officerID: Joi.string().regex(/^[a-zA-Z0-9_-]*$/).required(),
 	siteCode: Joi.number().integer().required(),
 	inPenaltyGroup: Joi.boolean().required(),
 };
 
 export default {
 	request: Joi.object().keys({
-		ID: Joi.string().required(),
-		Hash: Joi.string().required(),
+		ID: Joi.string().token().required(),
+		Hash: Joi.string().required().alphanum(),
 		Enabled: Joi.boolean().required(),
 		Offset: Joi.number(),
 		Value: Joi.object(valueSchema),
-		Origin: Joi.string(),
-		VehicleRegistration: Joi.string(),
+		Origin: Joi.string().valid(['APP']),
```

Comment:
I feel like individual penalty documents can have an `Origin` other than `APP` somehow. Probably worth some investigation, the purpose this field has always been a bit beyond me.

Reply:
I couldn't see any example of the Origin being anything else unless it was manually set during testing. I'll change it to allow A-Z to be safe.